app/controllers/content_change_requests_controller.rb

```diff
@@ -16,6 +16,7 @@ def parse_request_from_params
   def content_change_request_params
     params.require(:support_requests_content_change_request).permit(
       :title,
+      :reason,
```

Comment:
I notice all the other user-provided inputs have a corresponding [`LabelledSnippet` in the `ContentChangeRequestTicket` model](https://github.com/alphagov/support/blob/7012da5f0f798a357b6429b9eaeb6544abb77927/app/models/zendesk/ticket/content_change_request_ticket.rb#L18) - do we not need that? Is it because the others are text inputs and this one is a dropdown?

EDIT: I notice we also don't add it for the new `why_is_change_needed` field, and that one is a text input 🤔 

EDIT EDIT: ahh, you add them in `Updates test with new fields`. I think you should fix these up to the commits in which you add the fields - makes it easier to review when every change is self-contained 👍